Yona/Yona/nl.lproj/Localizable.strings

```diff
@@ -6,6 +6,10 @@
  Copyright © 2016 Yona. All rights reserved.
  */
 
+//Fabric API key file
+"fabric-alert-title" = "Intialisatiefout";
+"fabric-failure" = "Ontbrekende Fabric API-sleutel";
```

Comment:
Say "API-key", as this is normally not translated in Dutch.

Reply:
updated